src/joinRates/columns.tsx

```diff
@@ -1,14 +1,17 @@
+import { renderNumericCell } from '../profiler/columns'
+
 export const getJoinRatesColumns = (columns: Array<string>) =>
   columns.map(column => {
     return {
       id: column,
       title: column,
       dataIndex: column,
-      render: (data: any) => {
-        if (data && column === 'Table names') {
+      render: (data: Array<string> | number) => {
+        if (data && Array.isArray(data) && column === 'Table names') {
           return data.join(', ')
         }
-        return data
+
+        return renderNumericCell(data as number)
```

Comment:
Don't we need to check data also for renderNumericCell?
`if (data) {
    return renderNumericCell(data as number)
}`
I'm not sure but in case there is no data would we have a page crash on the `renderNumericValue` function error?